agir/events/models.py

```diff
@@ -1048,3 +1048,60 @@ def __str__(self):
 
     class Meta:
         verbose_name = "Visio-conférence"
+
+
+class Invitation(TimeStampedModel):
+
+    INVITATION_PENDING = "pending"
+    INVITATION_ACCEPTED = "accepted"
+    INVITATION_REFUSED = "refused"
+
+    STATUSES = (
+        (INVITATION_PENDING, "En attente"),
+        (INVITATION_ACCEPTED, "Acceptée"),
+        (INVITATION_REFUSED, "Refusée"),
+    )
+
+    person_request = models.ForeignKey(
+        "people.Person",
+        on_delete=models.CASCADE,
+        verbose_name="Personne qui émet l'invitation",
+    )
+    person_response = models.ForeignKey(
+        "people.Person",
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+        related_name="invitation_response",
+        verbose_name="Personne qui répond à l'invitation",
+    )
```

Comment:
C'est peut-être plus clair `sender` et `recipient` ?
```suggestion
    sender = models.ForeignKey(
        "people.Person",
        on_delete=models.CASCADE,
        verbose_name="Personne qui émet l'invitation",
    )
    recipient = models.ForeignKey(
        "people.Person",
        on_delete=models.CASCADE,
        null=True,
        blank=True,
        related_name="invitation_response",
        verbose_name="Personne qui répond à l'invitation",
    )
```